AudicaMod/src/Hooks.cs

```diff
@@ -341,5 +341,130 @@ private static void Prefix(SongListControls __instance)
             }
         }
 
+        [HarmonyPatch(typeof(SongList), "GetSong", new Type[] { typeof(string) })]
+        private static class AddToScrollerPatch
+        {
+            private static void Postfix(SongList __instance, string songID, SongList.SongData __result)

```

Comment:
Is there no way to set these once after a song list has been loaded? `GetSong` gets called pretty frequently by the game. Having to do all these checks each time is really inefficient.

Reply:
I'll look at it today, I probably can make it happen right after the song list loaded from game boot, but I'm not sure how to detect when the list refresh is complete from the mod.

Reply:
This is fixed.